frontend_tests/node_tests/image_upload_widget.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+
+zrequire("ui_report");
+set_global("csrf_token", "csrf_token");
+set_global("$", global.make_zjquery());
+
+const Image_upload_widget = zrequire("image_upload_widget");
```

Comment:
This variable name is not consistent with our style.  I think if you're doing a class you should call it `ImageUploadWidget`.

Also, probably we should just make the new file an ES6 class like `static/js/fold_dict.ts`